CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ---
 
+## [0.14.6] - 2023-08-24
+
+### Changed
```

Comment:
I would say this is a fix, not a change, since the library didn't compile without it

Reply:
Makes sense